recipes/rply/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "rply" %}
+{% set version = "0.7.4" %}
+{% set sha256 = "723303d6c5f05a7ee19f59531f66c8c7d41cfaef2676057369db1eb5520b378b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - appdirs
+  run:
+    - python
+
+test:
+  imports:
+    - rply
+
+about:
+  home: https://github.com/drewokane/rply
```

Comment:
I noticed that this is repo is a fork of https://github.com/alex/rply and it does not seem to contain any further updates. Can you explain the reason for the fork and why it should be the basis of this recipe vs the original repository? The pypi index is also owned by Alex.